pkgs/tools/misc/webcat/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  pname = "webcat";
+  version = "unstable-20210906";
+
+  src = fetchgit {
+    url = "https://codeberg.org/rumpelsepp/webcat.git";
+    rev = "57a65558f0affac0b2f8f4831c52964eb9ad5386";
+    sha256 = "15c62sjr15l5hwkvc4xarfn76341wi16pjv9qbr1agaz1vqgr6rd";
+  };
+
+  vendorSha256 = "1apnra58mqrazbq53f0qlqnyyhjdvvdz995yridxva0fxmwpwcjy";
+
+  meta = with lib; {
+    homepage = "https://rumpelsepp.org/blog/ssh-through-websocket/";
+    description = "The lightweight swiss army knife for websockets";
+    license = licenses.gpl3Plus;
```

Comment:
are you sure this is gpl3Plus? I can't find any mentions of "any later version"

Reply:
To be honest, I don't know. I've followed the advice of @Pacman99 but you may be right. I will switch back to `gpl3Only`

Reply:
Yeah I'm not sure, I didn't see any text at the top of any of the source files. I'm not sure what we should default to if its not specified.

Reply:
the default would be gpl3Only